docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2'
```

Comment:
scripts to be able to easily run the tests without needing to rebuild the docker image